nrich-search/README.md

```diff
@@ -39,31 +44,48 @@ public class ApplicationConfiguration {
         return new SearchRepositoryFactorySupportFactory(stringToEntityPropertyMapConverter);
     }
 }
-
 ```
 
-`StringToTypeConverter<Object>` performs conversion from string to typed instances and is used when querying registry entities. Default implementation (`DefaultStringToTypeConverter`)
-accepts a list of data formats and regular expressions that are used to convert string to type found in properties of entity classes.
+### `StringToTypeConverter<Object>`
+
+`StringToTypeConverter<Object>` performs conversion from string to typed instances and is used when querying registry entities.
+Default implementation (`DefaultStringToTypeConverter`) accepts a list of data formats and regular expressions that are used to
+convert string to the type found in properties of entity classes.
+
+---
+
+### `StringToEntityPropertyMapConverter`
 
-`StringToEntityPropertyMapConverter` is also used for querying registry entities. It is responsible for creating conditions from query string and a list of properties to search (conversion to typed
-instances is delegated to `StringToTypeConverter<?>`). When searching using `StringSearchExecutor` it accepts a query string, and a list of properties to be searched that are then
-converted using  `StringToEntityPropertyMapConverter` to properties.
+`StringToEntityPropertyMapConverter` is also used for querying registry entities.
+It is responsible for creating conditions from query string and a list of properties to search (conversion to typed instances is delegated to `StringToTypeConverter<?>`).
+When searching using `StringSearchExecutor` it accepts a query string, and a list of properties to be searched that are then converted using
```

Comment:
"...that are then converted to properties using `StringToEntityPropertyMapConverter` ." sounds better.

Reply:
It does!